### DISCUSSION TITLE
Configuring Amazon SES transactional service I get error 412 SES: The request signature we calculated does not match the signature you provided

### DISCUSSION THREAD

**Discussion by danieldmu** [emailjs-com/help#134] [General]
The problem was the AWS user must NOT be created from the SES Console (that produces a SMTP user), for the service to work the AWS user must be created from the AIM console (with the required ses:SendRawEmail access). 

That solved my issue, posting it here for anyone who has the same issue.

**Comment by techgaun** [comment]
This is interesting because I've used the SMTP user created through SES console in places and they have worked. Seems emailjs specific? But thank you for theinfo. This was helpful. Would be good to specify this in emailjs docs
